ggml-cuda.cu

```diff
@@ -173,6 +206,41 @@ static __global__ void dequantize_block_q8_0(const void * vx, float * y) {
     }
 }
 
+template <int block_size, int qk, dequantize_kernel_t dequantize_kernel> static __global__ void dequantize_mul_mat_vec(const void * vx, const float * y, float * dst, const int ncols) {
```

Comment:
Could the `template<...>` be on a separate line? It's getting very long otherwise.